client/src/routes/+layout.svelte

```diff
@@ -177,6 +194,10 @@
     left: 8px;
   }
 
+  h3 {
+    margin-block: 10px;
```

Comment:
this feels a bit cramped on large screens.

we tried only changing the `margin-top` which looked kinda good

we also considered using the compact styles only on narrow devices

I think more discussion is needed

Reply:
after playing around with both, i prefer the margin-bottom as well.